pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -100,6 +102,70 @@ def test_rpm(self):
         )
         self.assertDictEqual(get_added_content_summary(repo), {})
 
+    @unittest.skip("Kickstart fixture with wrong checksums")
```

Comment:
I had to put this because the fixture seems to be with the wrong checksums:
https://repos.fedorapeople.org/pulp/pulp/fixtures/rpm-kickstart/.treeinfo

Reply:
LiveOS/squashfs.img
- current = 158657d4ff270db50204ae70c2256ca1523ee4f3c23eeedbabc3f723b69163a2
- calculated = 30e14955ebf1352266dc2ff8067e68104607e750abb9d3b36582b8af909fcb58

images/pxeboot/initrd.img
- current = 094631a67f1766688dfcf4ec1f07ab0921100878736afc3cc1b31f69e7b517a1
- calculated = 30e14955ebf1352266dc2ff8067e68104607e750abb9d3b36582b8af909fcb58

images/pxeboot/upgrade.img
- current = bc884c1b4b520e57f81f3bf43156365d9ef651a47266b21e2c3d00b09e0da46b
- calculated = 30e14955ebf1352266dc2ff8067e68104607e750abb9d3b36582b8af909fcb58

images/pxeboot/vmlinuz
- current = 91052b444e73f3eebdb93d1fb1506597e96c92d8de9c1e3c3f36b07a57d0a18f
- calculated = 91052b444e73f3eebdb93d1fb1506597e96c92d8de9c1e3c3f36b07a57d0a18f

Reply:
I agree the checksums are incorrect. In my own testing I found this also:

```
[bmbouter@localhost pulp_ansible]$ sha256sum ~/Downloads/squashfs.img 
30e14955ebf1352266dc2ff8067e68104607e750abb9d3b36582b8af909fcb58  /home/bmbouter/Downloads/squashfs.img
```